### ISSUE TITLE
[Bug] Pixels Overflow on Join Organization Page

### ISSUE THREAD

**Issue by TusharGupta05** [PalisadoesFoundation/talawa#336]
As it is visible in the screenshot, on the join organization page, pixels are overflowed.
**Steps to reproduce :** Run the app on iPhone 5

![Screenshot (212)](https://user-images.githubusercontent.com/56271252/111229347-c141a880-860b-11eb-815e-9afe94405e21.png)

I would like to work on this. @mentors please assign this issue to me.
Also, I am on the master branch. So, I don't think it is a duplicate.

**Comment by ram2939** [comment]
I would like to work on this issue

**Comment by Sagar2366** [comment]
Have you tested this scenario for Android phone as well ? Most of User base will be Android users only. So we might not run into this condition.

**Comment by TusharGupta05** [comment]
@Sagar2366 sir, I have tested it on 10 different screen sizes. 3 of them had the same problem. And 2 of those three are android phone screen sizes. I have attached the screenshots of both of the android phones.
![Screenshot (212)](https://user-images.githubusercontent.com/56271252/111405335-4c976880-86f6-11eb-9dc3-548168da97fd.png)
![Screenshot (216)](https://user-images.githubusercontent.com/56271252/111405355-528d4980-86f6-11eb-9d57-43a910c1acd1.png)
